iib/web/models.py

```diff
@@ -1006,13 +1035,21 @@ def from_json(cls, kwargs, batch=None):
             raise ValidationError(f'"operators" should be a non-empty array of strings')
 
         cls._from_json(
-            request_kwargs, additional_required_params=['operators', 'from_index'], batch=batch
+            request_kwargs,
+            additional_required_params=['operators', 'from_index'],
+            batch=batch,
```

Comment:
If you remove the last comma, it will go back to as it was before. I'm guessing this change was unintentional and Python black formatting.